src/SearchParams.jsx

```diff
@@ -45,15 +46,22 @@ const SearchParams = () => {
         ) : null}
         <label htmlFor="location">
           Location
-          <input id="location" name="location" placeholder="Location" />
+          <input
+            type="text"
```

Comment:
we should define the types of the input if we want to get default input styles from tailwind form plugin